packages/telemetry/src/frcat-entrypoint.js

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env node
+/* global process, BigUint64Array */
+// frcat - print out the contents of a flight recorder
+
+import '@endo/init';
+
+import { makeMemoryMappedCircularBuffer } from './flight-recorder.js';
+
+const main = async () => {
+  const files = process.argv.slice(2);
+  if (files.length === 0) {
+    throw Error('no flight-recorder.bin files specified');
+  }
+
+  for await (const file of files) {
+    const { readCircBuf } = await makeMemoryMappedCircularBuffer({
+      circularBufferFile: file,
+      circularBufferSize: null,
+    });
+
+    let offset = 0;
+    for (;;) {
+      const lenBuf = new Uint8Array(BigUint64Array.BYTES_PER_ELEMENT);
+      const { done } = readCircBuf(lenBuf, offset);
+      if (done) {
+        break;
+      }
+      offset += 8;
```

Comment:
This `8` sticks out as a magic constant, I think everywhere else you've used some buffer math to compute the size, should this be computed too? Like, the length of `lenBuf`?